lnrpc/routerrpc/router_backend.go

```diff
@@ -192,6 +201,7 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 			)
 		},
 		DestPayloadTLV: len(destTLV) != 0,
+		CltvLimit:      cltvLimit,
```

Comment:
Should the final cltv delta be substracted from this? I think it may be a pre-existing bug, but now that the limit is enforced hard by the outgoing link, it may be more important to fix.

Reply:
Nice catch, fixed.